lib/workers/repository/process/index.ts

```diff
@@ -76,7 +76,7 @@ async function getBaseBranchConfig(
     baseBranchConfig.baseBranches = config.baseBranches;
   }
 
-  if (config.baseBranches!.length > 1) {
+  if (config.baseBranchPatterns!.length > 1) {
```

Comment:
Does this PR supersede #35423, or would it be rebased/merged and applied still?

Reply:
This will be merged first